build/generate-emoji.go

```diff
@@ -102,18 +106,20 @@ func generate() ([]byte, error) {
 		return nil, err
 	}
 
-	var re = regexp.MustCompile(`keycap|registered|copyright`)
-	tmp := data[:0]
+	var skinTones = make(map[string]string)
 
-	// filter out emoji that require greater than max unicode version
+	skinTones["🏻"] = "Light Skin Tone"
```

Comment:
Probably best to use \uxxxx notation for these

Reply:
done